rd_ui/app/scripts/filters.js

```diff
@@ -49,9 +49,12 @@ angular.module('redash.filters', []).
   .filter('colWidth', function () {
     return function (widgetWidth) {
       if (widgetWidth == 1) {
-        return 6;
+        return 4;
+      } else if (widgetWidth == 2) {
+        return 8;
+      } else {
+        return 12;
       }
-      return 12;
```

Comment:
Also, looks like the gridster preview wasn't updated for the new dimensions: 

![image](https://cloud.githubusercontent.com/assets/71468/10417026/94295280-7036-11e5-83bc-f9d9004a3d9a.png)